Tests/Remora.Discord.Tests/Samples/Objects/MESSAGE_SNAPSHOT/MESSAGE_SNAPSHOT.json

```diff
@@ -0,0 +1,45 @@
+{
+  "message": {
```

Comment:
Please add a separate file where the partial message is an empty object (`MESSAGE_SNAPSHOT.partials.json`) - we want to cover both a populated and unpopulated message path.